src/pg/sql/42_observatory_exploration.sql

```diff
@@ -0,0 +1,30 @@
+
+-- return a table that contains a string match based on input
+-- TODO: implement search for timespan
+
+CREATE OR REPLACE FUNCTION OBS_SearchTables(
+  search_term text,
+  time_span text DEFAULT '2009 - 2013'
```

Comment:
Since this is not census-specific, this should default to `NULL`.  Then, there should be logic in the function that handles the NULL case, most likely by returning everything regardless of timespan, and having a second column in the return for time_span.